src/checkstatus/SSMAutomationExecutionStatusWaiter.py

```diff
@@ -0,0 +1,31 @@
+import botocore
+import constants
+import waiter_acceptor_config
+
+def check_automation_execution_status(automation_execution_id, max_attempts, ssm):
+    acceptor_config = waiter_acceptor_config.SSM_AUTOMATION_EXECUTION_COMPLETED
+    model = botocore.waiter.WaiterModel({
+        "version": 2,
+        "waiters": {
+            "SSMAutomationExecutionStatus": {
+                "delay": constants.DELAY_INTERVAL,
+                "operation": "DescribeAutomationExecutions",
+                "maxAttempts": max_attempts,
+                "acceptors": acceptor_config
+            }
+        }
+    })
+
+    automation_execution_filter = [
+        {
+            'Key': 'ExecutionId',
+            'Values': [
+                automation_execution_id,
```

Comment:
Does an incorrect execution id lead to exceptions ?

Reply:
the response of DescribeAutomationExecutions would be empty